src/components/BareMetalHosts/fixtures/BaremetalHostStatus.fixture.js

```diff
@@ -27,4 +27,43 @@ export default [
       },
     },
   },
+  {
+    component: BaremetalHostStatus,
+    name: 'Show progress state for `match profile` status',
+    props: {
+      host: {
```

Comment:
please can you create a helper function which creates this structure?

Reply:
Done